protocol/avdl/chat1/common.avdl

```diff
@@ -187,6 +187,12 @@ protocol common {
     bytes k; // Verifying key
   }
 
+  record SignEncryptedData {
```

Comment:
Why make a new struct for this as opposed to just using `EncryptedData`? Just so the name is clearer?